### DISCUSSION TITLE
4th dose vaccinations

### DISCUSSION THREAD

**Discussion by Mike-Honey** [owid/covid-19-data#2617] [Ideas]
Several countries (Israel, Australia, others) are already well underway delivering 4th dose vaccinations. Can you add a column for this to the dataset and start integrating the available data?

You might save some time later by also adding columns for 5th dose, 6th dose, 7th dose etc.

**Comment by lucasrodes** [comment]
hi @Mike-Honey,
All additional doses are counted together under the metric `total_boosters`.

**Reply by Mike-Honey** [reply]
Hi @lucasrodes 

Thank you for your work on this amazing dataset, and thanks for the speedy reply.

Thanks for clarifying on that question. I'd like to count people (or % of population) with different number of doses (1st, 2nd, 3rd, 4th etc), but it seems that's not possible from this data (beyond 1st & 2nd).

**Reply by lucasrodes** [reply]
Hi @Mike-Honey,
Due to our limited resources, we are not planning to report metrics relative to _n_-doses anytime soon.

Sorry for the inconvenience!
